src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -15,9 +20,13 @@ public class CommandBox extends UiPart<Region> {
 
     public static final String ERROR_STYLE_CLASS = "error";
     private static final String FXML = "CommandBox.fxml";
+    private static final int ZERO_INDEX = 0;
+    private static final Logger logger = LogsCenter.getLogger(CommandBox.class);
```

Comment:
Good user of Logger right here!

Am unsure of how Logger works but following how you did this implementation, I am clearer as to how to use it in my snippet of code as well 👍🏻